mono/profiler/proflog.c

```diff
@@ -217,20 +217,24 @@ typedef struct _LogBuffer LogBuffer;
  *
  * type metadata format:
  * type: TYPE_METADATA
- * exinfo: flags: TYPE_LOAD_ERR
+ * exinfo: one of: TYPE_END_LOAD, TYPE_END_UNLOAD (optional for TYPE_THREAD and TYPE_DOMAIN)
```

Comment:
We've never actually emitted this flag so I decided to remove it in order to not mislead consumers of the format into thinking they'll actually get it and therefore relying on it.

Hmm. I should have done this in a separate commit, and we should probably mention it in the format changelog even if it's not technically a format change.